static/app/utils/api/useFetchParallelPages.tsx

```diff
@@ -127,7 +125,7 @@ export default function useFetchParallelPages<Data>({
 
             const [data, , resp] = await queryClient.fetchQuery({
               queryKey: getQueryKey({cursor, per_page: perPage}),
-              queryFn: fetchDataQuery(api),
+              queryFn: fetchDataQuery<Data>,
```

Comment:
`fetchDataQuery` can now be invoked directly - it doesn’t have to be a function that returns a function. Also, it now defaults to `unknown` instead of `any` for data that is returned, but it accepts a type parameter to specify the return type. This helps us not accidentally leaking `any` around.